packages/fetch-router/demos/node/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "fetch-router-node-demo",
+  "private": true,
+  "type": "module",
+  "dependencies": {
+    "@remix-run/cookie": "workspace:*",
+    "@remix-run/fetch-router": "workspace:*",
+    "@remix-run/html-template": "workspace:*",
+    "@remix-run/node-fetch-server": "workspace:*",
+    "@remix-run/route-pattern": "workspace:*"
+  },
+  "devDependencies": {
+    "@types/node": "^24.6.0"
+  },
+  "scripts": {
+    "dev": "node --watch server.ts",
+    "start": "node server.ts",
```

Comment:
I opted to just use node since it automatically strips types these days. Let me know if you want me to use `tsx` for any reason